modules/core/example/ts/create-wallet-addresses.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Create a new receive address on a multi-sig wallet at BitGo.
+ *
+ * Copyright 2019, BitGo, Inc.  All Rights Reserved.
+ */
+import { BitGo } from '../../src';
+const bitgo = new BitGo({ env: 'test' });
+
+const coin = 'tltc';
+const basecoin = bitgo.coin(coin);
+// TODO: set your access token here
+const accessToken = '';
+// TODO: set your passphrase here
+const walletPassphrase = '';
+
```

Comment:
same here, i'd throw an error or add a warning saying "make sure you add your access token / passphrase to the script"